docs/content/Getting-Started/Overview.mdx

```diff
@@ -7,6 +7,8 @@ menuOrder: 1
 isDisableFeedbackBlock: false
 ---
 
+Cube is a cloud-native application, packaged and distributed as Docker images and designed to run in a containerized environment.
```

Comment:
Could maybe also add a link to https://docs.docker.com/get-started/overview/#images?

Reply:
I suggest we omit this link here but maybe add it later to the Architecture page. Would be strange if we link to Docker's website first thing in Getting started.

Reply:
Fair enough 😀